packages/styles/scss/patterns/sub-patterns/quote/_quote.scss

```diff
@@ -11,11 +11,19 @@
 
 @mixin quote {
   .#{$prefix}--quote {
+    padding-bottom: 2rem;
```

Comment:
@andysherman2121 can you use spacing tokens here instead?

Reply:
Oh yeah good catch,  i'll update that